types/seeed/senseCAPT1000A/settings.schema.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$id": "https://akenza.io/seeed/senseCAPT1000A/settings.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "settings",
+  "title": "Settings",
+  "properties": {
+    "motionEnabled": {
+      "title": "Motion enabled",
+      "type": "integer",
+      "description": "Motion enabled",
+      "hideFromKpis": true
+    },
+    "anyMotionThreshold": {
+      "title": "Motion threshold",
+      "type": "integer",
+      "description": "Motion threshold",
+      "hideFromKpis": true
+    },
+    "motionStartInterval": {
+      "title": "Motion start interval",
+      "type": "integer",
+      "unit": "min",
+      "description": "Motion start interval",
+      "hideFromKpis": true
+    },
+    "staticEnabled": {
+      "title": "Static enabled",
+      "type": "integer",
+      "description": "Static enabled",
+      "hideFromKpis": true
+    },
+    "deviceStaticTimeout": {
+      "title": "Device static timeout",
+      "type": "integer",
+      "unit": "min",
+      "description": "Device static timeout",
+      "hideFromKpis": true
+    },
+    "shockEnabled": {
+      "title": "Shock enabled",
+      "type": "integer",
+      "description": "Shock enabled",
+      "hideFromKpis": true
+    },
+    "shockThreshold": {
+      "title": "Shock threshold",
+      "type": "integer",
+      "description": "Shock threshold",
+      "hideFromKpis": true
+    },
+    "tempEnabled": {
```

Comment:
align the use of abbreviations, I would general refrain from it to stay consistent with the measurement